schema.json

```diff
@@ -1686,8 +1634,12 @@
             "type": "object"
         },
         "region": {
-            "description": "A region.",
+            "$ref": "#/definitions/simplestring",
+            "description": "A region."
+        },
+        "simplestring": {
             "minLength": 1,
+            "pattern": "^(\\S+)( \\S+)*$",
```

Comment:
Played around with this on [regex101](https://regex101.com/r/SyW84W/1) (without the escaping backslashes, in ECMAScript regex which is the flavour JSON schema uses), and this seems to work fine.